pkg/abstractions/image/build.go

```diff
@@ -72,17 +74,61 @@ func (o *BuildOpts) String() string {
 	fmt.Fprintf(&b, "  \"BaseImageName\": %q,", o.BaseImageName)
 	fmt.Fprintf(&b, "  \"BaseImageTag\": %q,", o.BaseImageTag)
 	fmt.Fprintf(&b, "  \"BaseImageCreds\": %q,", o.BaseImageCreds)
+	fmt.Fprintf(&b, "  \"ExistingImageUri\": %q,", o.ExistingImageUri)
+	fmt.Fprintf(&b, "  \"ExistingImageCreds\": %#v,", o.ExistingImageCreds)
+	fmt.Fprintf(&b, "  \"Dockerfile\": %q,", o.Dockerfile)
+	fmt.Fprintf(&b, "  \"BuildCtxObject\": %q,", o.BuildCtxObject)
```

Comment:
What exactly is in this var?

Reply:
The object ID of the synced build context directory